frontend/web/components/pages/project-settings/tabs/ImportTab.tsx

```diff
@@ -0,0 +1,41 @@
+import ImportPage from 'components/import-export/ImportPage'
+import InfoMessage from 'components/InfoMessage'
+import React from 'react'
+import { useGetProjectQuery } from 'common/services/useProject'
+
+type ImportTabProps = {
+  projectId: number
+  environmentId?: string
+}
+
+export const ImportTab = ({ environmentId, projectId }: ImportTabProps) => {
+  const { data: project, isLoading } = useGetProjectQuery({
+    id: String(projectId),
+  })
```

Comment:
Wdyt about passing these as props and avoid this hook ? Not strongly opiniated as this is cached.

But deep down, the isLoading should (imo) be handled in the parent and keep this one as presentational as possible. Especially as if there is no project, this component does make sense